lib/msf/core/rpc/v10/client.rb

```diff
@@ -69,10 +69,10 @@ def call(meth, *args)
     res = @cli.send_recv(req)
     @cli.close
 
-    if res and [200, 401, 403, 500].include?(res.code)
+    if res && [200, 401, 403, 500].include?(res.code)
       resp = MessagePack.unpack(res.body)
 
-      if resp and resp.kind_of?(::Hash) and resp['error'] == true
+      if resp && resp.kind_of?(::Hash) && resp['error']
```

Comment:
Yup, sorry for not being more clear.

Pro returns stuff across the RPC bridge like:

```ruby
{ "error" => "" }
```

Or 

```ruby
{ "error"=> "you're doing it wrong" }
```

which trigger exceptions where they didn't before.  One could argue that pro shouldn't use the `error` key here, but that is a different fix.